lighthouse-core/audits/accessibility/axe-audit.js

```diff
@@ -76,14 +76,8 @@ class AxeAudit extends Audit {
     if (rule && rule.nodes) {
       items = rule.nodes.map(axeNode => ({
         node: {
-          type: /** @type {'node'} */ ('node'),
-          lhId: axeNode.node.lhId,
-          selector: axeNode.node.selector,
-          path: axeNode.node.devtoolsNodePath,
-          snippet: axeNode.node.snippet,
-          boundingRect: axeNode.node.boundingRect,
+          ...Audit.makeNodeValue(axeNode.node),
           explanation: axeNode.failureSummary,
```

Comment:
Why do we add `explanation` here?

Reply:
Doesn't look like this gets included in the report 🤔 . Is it supposed to?

Reply:
Only purpose I can see is in the smoke tests.